### PR TITLE
Reorganize form wizard and use system drafts for forms

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/blocks/form/formEditBehaviorSettings/formEditBehaviorSettings.ejs [hunk 1]
```diff
@@ -12,7 +12,7 @@
                     collections: this.collections,
                     mark: 'form-show-conditions',
                     models: {
-                        form: this.models.form
+                        popup: this.models.form.get('presentation') || this.models.form.createPresentation()
```
Comment (Radiansz) [inline src/blocks/form/formEditBehaviorSettings/formEditBehaviorSettings.ejs]: This string and next ones like it in this file. If there is no presentation in form, three instanses of presentation will be created. May be carry out this string up in to variable?

File: src/blocks/wizard/__step/formEditBehaviorSettings/formEditBehaviorSettings.js [hunk 2]
```diff
@@ -3,7 +3,15 @@ define(function(require) {
     var Step = require('blocks/wizard/__step/__step');
 
     return Step.extend({
-        template: require('tpl!./formEditBehaviorSettings.ejs')
+        template: require('tpl!./formEditBehaviorSettings.ejs'),
+
+        activate: function() {
```
Comment (Radiansz) [inline src/blocks/wizard/__step/formEditBehaviorSettings/formEditBehaviorSettings.js]: Better to listen on change form presentation

File: src/blocks/form/formEditBehaviorSettings/formEditBehaviorSettings.ejs (hunk 1 above)
Comment (zouxuoz) [inline src/blocks/form/formEditBehaviorSettings/formEditBehaviorSettings.ejs]: fixed

File: src/blocks/wizard/__step/formEditBehaviorSettings/formEditBehaviorSettings.js (hunk 2 above)
Comment (zouxuoz) [inline src/blocks/wizard/__step/formEditBehaviorSettings/formEditBehaviorSettings.js]: fixed

**Comment by Radiansz** [comment]
lgtm

**Pull request merged**